### ISSUE TITLE
Connection Per Host is not working

### ISSUE THREAD

**Issue by cmandesign** [ftpgrab/ftpgrab#47]
Dears, 
First of all thanks for developing such a simple and reliable tool. Second, I want to know why the connection_per_host parameter is not working at all? 
I've set the parameter to 10, but it only transfer 1 file simultaneously. I want to make it faster with multiple connections, what should i do now ?

```
ftp:
  host: 127.0.0.1
  port: 21
  username: test
  password:
  connections_per_host: 10
  timeout: 5
  disable_epsv: false
  tls:
    enable: false
    implicit: true
    insecure_skip_verify: false
  sources:
    - /Src

db:
  enable: true
  path: update.db

download:
  output: Dest\
  uid:
  gid:
  include:
  exclude:
  since: 0001-01-01T00:00:00Z
  retry: 3
  hide_skipped: false
  create_basedir: false
```

**Comment by crazy-max** [comment]
Hi @cmandesign, can you open a proper [bug report](https://github.com/ftpgrab/ftpgrab/issues/new?template=bug_report.md) please ? Thanks for your support!

**Comment by crazy-max** [comment]
Closed by #48
